lib/Partitioner/PartitionerUtils.cpp

```diff
@@ -140,72 +140,154 @@ getOutUsersWithOnePredecessor(const std::set<Node *> &nodes) {
   return ret;
 }
 
+/// \returns the list of users who use \p node's \p idx - th output.
+std::vector<Node *> getNodeUsers(Node *node, size_t idx) {
```

Comment:
This seems like a useful helper. Do you know if we do it elsewhere? If so perhaps it should be located somewhere more central so it can be reused in those places? Maybe even inside NodeValue.

Reply:
I will check it. If it is necessary, I will create another PR for this function:)